api/src/main/java/org/eclipse/microprofile/metrics/Snapshot.java

```diff
@@ -64,7 +64,9 @@ public abstract class Snapshot {
      *
      * @return an array of {@link HistogramBucket} if it is available or an empty array if not available
      */
-    public abstract HistogramBucket[] bucketValues();
+    public HistogramBucket[] bucketValues() {
+        return null;
```

Comment:
Shouldn't the default implementation return `new HistogramBucket[0]` to conform to the Javadoc?

Also, I realize that there is normally not much of a need for unit tests because the TCK covers the testing needs. But in this case would it make sense to add a unit test class that extends `Snapshot` but does not have its own implementation of this method? Simply trying to compile the class during a normal build would verify that the default implementation on `Snapshot` is in place.